templates/learn/get-started.hbs

```diff
@@ -46,13 +46,13 @@
     </header>
     {{#fluent "learn-generating-steps"}}
     {{#fluentparam "tree"}}
-    <pre><code>hello-rust
+    <pre><code class="plaintext">hello-rust
 |- Cargo.toml
 |- src
   |- main.rs</code></pre>
     {{/fluentparam}}
     {{#fluentparam "output"}}
-    <pre><code>$ cargo run
+    <pre><code class="plaintext">$ cargo run
```

Comment:
I tried using the `console` and `shell` language classes, which are supposedly [supported](https://github.com/highlightjs/highlight.js/blob/fc33aea76518928f791589f01371908949c332aa/SUPPORTED_LANGUAGES.md) by Highlight.js, but I guess our build doesn't have them? It would be nice to use that instead so that it's not just a wall of text.

Reply:
@XAMPPRocky Do you think you could add Shell to our custom build?

Reply:
Seems sensible to me. Changing the custom build is easy and I think would make sense as part of the PR. You can download a new one here, just uncheck all the unwanted languages, and replace the static files. https://highlightjs.org/download/

Reply:
Hmm, do we still need YAML support?

Reply:
Probably not.

Reply:
Okay then, I think I'll do:

* shell session
* plaintext
* rust
* bash
* toml
* markdown

Reply:
Only the `$` was syntax highlighted 😄:

![image](https://user-images.githubusercontent.com/37223377/92165625-2f4f1c80-edec-11ea-8f26-a1dc8ab2fe10.png)